nicegui/static/templates/optional_packages.html

```diff
@@ -0,0 +1,51 @@
+{% if options.highcharts %}
+    {% include 'highcharts.html' %}
+{% endif %}
+
+{% if options.aggrid %}
+    {% if not options.no_internet %}
+        <script src="https://unpkg.com/ag-grid-community/dist/ag-grid-community.min.js"></script>
+    {% else %}
+        <script src="/templates/local/ag-grid-community.js"></script>
```

Comment:
Here are still some URLs with leading "/". Probably not too important, since NiceGUI isn't using aggrid, but we could fix it nevertheless.